structs/student_validator.c

```diff
@@ -0,0 +1,46 @@
+#include<stdio.h>
+
+typedef struct 
+{
+    int age;
+    int birth;
+    char name[10];
+} StudentDate;
+
+int checkCard(StudentDate card) 
+{
+    int age = card.age;
+    int birth = card.birth;
+
+    return age < 18 || birth > 1991;
+}
+
+StudentDate feedBack(StudentDate card)
+{
+    int age = card.age;
+    int birth = card.birth;
+
+    if(checkCard(card))
+    {
+        printf("is not for you");
+    } else {
+        printf("Nice");
+    }
+}
+
+int main()
+{
+    int getAge;
+    int getBirthDay;
+    char getName[10];
+    
+    scanf("%d", &getAge);
+    scanf("%d", &getBirthDay);
+    scanf("%s", &getName);
+
+    StudentDate card = {.age = getAge, .birth = getBirthDay};
```

Comment:
You forgot to initialize the `name` field. You aren't using it either. So initialize it properly and suggest its application or get rid of it.